ingest/ingest_pipeline.py

```diff
@@ -351,20 +351,21 @@ def ingest_cell_metadata(self):
                     metadataModel.annot_header,
                 )
                 if status != 0:
-                    self.error_logger.error(
-                        f"Loading cell metadata header : {metadataModel.annot_header} failed. Exiting program",
-                        extra=self.extra_log_params,
+                    IngestPipeline.dev_logger.error(
+                        f"Loading cell metadata header : {metadataModel.annot_header} failed. Exiting program"
+                    )
+                    IngestPipeline.user_logger.error(
+                        f"Loading cell metadata header : {metadataModel.annot_header} failed. Exiting program"
                     )
                     return status
             return status if status is not None else 1
         else:
             report_issues(self.cell_metadata)
-            self.error_logger.error(
-                f"Cell metadata file format invalid", extra=self.extra_log_params
-            )
+            IngestPipeline.dev_logger.error("Cell metadata file format invalid")
+            IngestPipeline.user_logger.error("Cell metadata file format invalid")
```

Comment:
This kind of duplication is significant -- let's try to eliminate it.

All messages that go to `Foo.user_logger.error` should also go to `Foo.dev_logger.error`.  This should be possible in one statement -- `IngestPipeline.user_logger.error("Cell metadata file format invalid")` should add a new line with that message in both the `log.txt` and `user_log.txt` files.

This might be possible by refactoring `setup_logger` to accept an array of strings (`log_files` instead of `log_file`).  Then for each of those `log_file` strings, a new `handler` would be created and added via `logger.addHandler(handler)`.

Let's pair program if this isn't clear.  If it's easy, it'd be worthwhile.